scripts/ubuntu-20-provision.sh

```diff
@@ -191,6 +193,56 @@ function install_maven() {
   rm -f "/tmp/apache-maven-${MAVEN_VERSION}-bin.tar.gz"
 }
 
+## Ensure that hadolint is installed
+function install_hadolint() {
+  hadolint_arch_suffix="${ARCHITECTURE}"
+  if [ "${hadolint_arch_suffix}" == "amd64" ]; then
+    hadolint_arch_suffix="x86_64"
+  fi
+  curl --fail --silent --location --show-error --output /usr/local/bin/hadolint \
+    "https://github.com/hadolint/hadolint/releases/download/v${HADOLINT_VERSION}/hadolint-Linux-${hadolint_arch_suffix}"
```

Comment:
Does the same method as L182 for docker-compose be used here too?
If so,
```suggestion
  curl --fail --silent --location --show-error --output /usr/local/bin/hadolint \
    "https://github.com/hadolint/hadolint/releases/download/v${HADOLINT_VERSION}/hadolint-Linux-$(uname -m)"
```
(I suspect there is a reason you didn't do that)

Reply:
You suspect well ;) 

The output of `uname -m` on ARM64 CPUs is: `aarch64`

```bash
$ docker run --rm -ti --platform=linux/arm64 --entrypoint=uname ubuntu:20.04 -m 
aarch64
```

So problem here is that each tool have it's own usage (Ouptut of `GOOS` and `GOARCH` - https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63, output of `uname -m`, custom like what we are doing, etc.)

OItherwise I would have used the same variable everywhere